CONTRIBUTING.md

```diff
@@ -113,6 +113,38 @@ Then run the test cases:
 ```sh-session
 $ ./bin/test
 ```
+### Adding New Integration tests
+
+Each integration test exists in its own subdirectory within the "test"
+directory.  The test runner assumes it has the following executable scripts,
+which will be run in order:
+
+- `./start` (required) - Performs setup work -- eg, spinning up test
+   containers, populating a database with test data, etc.
+- `./test` (required) - Runs the actual tests.  It is assumed to produce go
+   test output on stdout.
+- `./stop` (optional) - Performs cleanup work.
+
+Hence, adding a new integration requires two steps:
+
+1. Creating a folder with test scripts as described above. 
+1. Adding a new entry to the `Jenkinsfile` to exercise it using the
```

Comment:
TIL to put a comma before "which" in nonrestrictive phrases.

More a suggestion than anything ...
I wonder if we can rephrase this so as to reduce the length of the sentence? It might make it simpler to parse.
For some reason the comma at `output file, which will` was throwing me off a bit.

Reply:
Sure, I can do that.